applicants.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/applicants.css">
+    <link rel="stylesheet" href="./src/styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <section>
+        <table>
+            <thead>
+                <tr>
+                    <td>Name</td>
+                    <td>Has feet</td>
+                    <td>Destination</td>
+                    <td>Gross feet</td>
+                </tr>
+            </thead>
+            <tbody id="applicants">
+                <tr>
```

Comment:
this section after the ```tbody``` tag does not need to be here because you are looping through all the applicants and dynamically appending the ```tr``` tags to the dom.